src/Component/QueryBuilderTag.tsx

```diff
@@ -0,0 +1,64 @@
+import { css } from '@emotion/css';
+import { GrafanaTheme } from '@grafana/data';
+import { getTagColorsFromName, stylesFactory, useTheme, IconButton } from '@grafana/ui';
+import React, { ComponentType } from 'react';
+
+type QueryBuilderTagProps = {
+  name: string;
+  value: string;
+  onRemove: (name: string) => void;
+};
+const getStyles = stylesFactory(({ theme, name }: { theme: GrafanaTheme; name: string }) => {
```

Comment:
@MicroOps-cn can you resolve deprecation of stylesFactory?

Reply:
@simPod It is caused by the name change of the request parameter and has been repaired.